src/components/search/UseSearch.jsx

```diff
@@ -1,11 +1,14 @@
 import { getClassesData } from '../../util/DataBaseRequests';
 import { useState, useEffect } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
 const useSearch = () => {
   const [classes, setClasses] = useState([]);
   const [currentPage, setCurrentPage] = useState(1);
   const [totalPages, setTotalPages] = useState(1);
   const [search, setSearch] = useState('');
   const [category, setCategory] = useState('');
+  const [searchParams, setSearchParams] = useSearchParams();
   const [loading, setLoading] = useState(false);
```

Comment:
Let's follow common naming conventions for boolean state.

Reply:
I  took it from here   https://reactrouter.com/en/main/hooks/use-search-params not sure if it's boolean

Reply:
For `loading`. `useSearchParams` is fine.